src/routes/feedRouter.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const { asyncWrap } = require("../utils/errorHandler");
+const { feedController } = require("../controllers");
+const feedRouter = express.Router();
+
+feedRouter.get('', asyncWrap(feedController.getFeedController));
+feedRouter.post('', asyncWrap(feedController.addFeedController));
+feedRouter.delete('/:feedId', asyncWrap(feedController.deleteFeedController));
+feedRouter.put('/:userId/:feedId/:imageId', asyncWrap(feedController.updateFeedController));
+feedRouter.delete('/:userId/:feedId/:imageId', asyncWrap(feedController.updateFeedController));
+feedRouter.get('/rank', asyncWrap(feedController.feedRankingController));
+
+module.exports = feedRouter;
```

Comment:
```suggestion
const express = require("express");
const { asyncWrap } = require("../utils/errorHandler");
const { feedController } = require("../controllers");
const feedRouter = express.Router();

feedRouter.get('', asyncWrap(feedController.getFeed));
feedRouter.post('', asyncWrap(feedController.addFeed));
feedRouter.delete('/:feedId', asyncWrap(feedController.deleteFeed));
feedRouter.put('/:feedId', asyncWrap(feedController.updateFeed));
feedRouter.delete(':feedId/:imageId', asyncWrap(feedController.updateFeed));
feedRouter.get('/rank', asyncWrap(feedController.getFeedRanking));

module.exports = feedRouter;
```

1. controller 함수 이름 뒤에 'controller' 붙이는 것은 팀내 컨벤션을 따르시면 될 것 같습니다. 현재 머지된 코드들에 controller가 뒤에 안붙어 있어서, 지우시면 좋을 것 같아요 :) 팀내 컨벤션이 변경되었다면 알려주세요!
2. delete메소드에 왜 updateFeedController가 연결되어 있나요??
3. userId는 uri에서 받아오시면 안되고, token에서 받아오셔야 합니다.
4. imageURL을 수정하시고자 한다면 request body에서 받아오시는 것이 좋을 것 같습니다.

Reply:
네! controller 수정하였고 , delete 메소드는 잘못 적은거로 지우며 수정했습니다!
userId는 넣어보고 해보았고, token값으로 받아오는걸로 수정하겠습니다.
imageURL req.body 변경완료하였습니다.